blocks/best-time-to-visit/render.php

```
@@ -0,0 +1,147 @@
+ * @return string CSS class name.
+ */
+function asnz_get_month_class($month_name, $best, $shoulder)
+{
+    if (in_array($month_name, $best, true)) {
+        return 'best-time-month--best';
+    } elseif (in_array($month_name, $shoulder, true)) {
+        return 'best-time-month--good';
+    } else {
+        return 'best-time-month--mixed';
+    }
+}
```

PHPMD (2.15.0)

The parameter $month_name is not named in camelCase. (undefined)

(CamelCaseParameterName)